### PR TITLE
Feature/pelagos 3736 create fileset entity

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: config/packages/doctrine.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ doctrine:
         # configure these for your database server
         driver: 'pdo_pgsql'
         charset: utf8
-        server_version: 9.6
+        server_version: 12
```
Comment (fightingtexasaggie) [inline config/packages/doctrine.yaml]: We probably should leave this at 9.6 as is checked in in code (stash it after making the migration), or will that make the migration not compatible?

Comment (praneethpr) [reply config/packages/doctrine.yaml]: Tried it, migration works at 9.6. But we should specify the correct version of postgres in the future

File: src/Entity/File.php [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    /**
+     * Doctrine identifier for File entity.
+     *
+     * @var integer
+     *
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    protected $id;
```
Comment (fightingtexasaggie) [inline src/Entity/File.php]: Are we overriding the base class Entity's ID?

File: src/Entity/File.php [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+    /**
+     * File path or location.
+     *
+     * @var string
+     *
+     * @ORM\Column(type="text", nullable=true)
+     */
+    protected $filePath;
+
+    /**
```
Comment (fightingtexasaggie) [inline src/Entity/File.php]: possibly not needed if we're using to base class getter/setter methods

File: src/Entity/Fileset.php [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+/**
+ * Fileset Entity class.
+ *
+ * @ORM\Entity(repositoryClass="App\Repository\FilesetRepository")
+ */
+class Fileset
+{
+
+    /**
```
Comment (fightingtexasaggie) [inline src/Entity/Fileset.php]: base class?

File: src/Entity/File.php (hunk 2 above)
Comment (praneethpr) [reply src/Entity/File.php]: I created the entity using bin/console make entity command, should've removed this. Thanks Mike

**Comment by fightingtexasaggie** [comment]
I put output of \d file \d fileset \d dataset_submission (trimmed) on the ticket for PO review is there is nothing to test.

File: src/Entity/File.php [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+{
+    /**
+     * Fileset which contains this file.
+     *
+     * @var Fileset
+     *
+     * @ORM\ManyToOne(targetEntity="Fileset", inversedBy="File")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    protected $fileset;
```
Comment (mickel1138) [inline src/Entity/File.php]: I don't think this relationship is needed, and may complicate things later.

**Pull request merged**